packages/members-api/lib/common.js

```diff
@@ -0,0 +1,28 @@
+function getLogging() {
+    let logging = {};
+    let _logger = null;
```

Comment:
Does this mean that the logger will be `null` everytime we try to use it before calling `updateLogger`?

Reply:
Yeah, the `updateLogger` is setup whenever `MembersApi()` is called, so ideally should be never, but we still handle `null` logger case if it comes up. Was thinking about a default logger case, but couldn't think of what that could be in production 🤷‍♂

Reply:
Oh sorry - I mean that everytime we do `require('path/to/common').logger` it will call this function again, resetting the logger to `null` :thinking:

Reply:
I pushed up a change for this :tada:

Reply:
🤦‍♂ Ah yes! Thanks 😄

Reply:
The change updates the default `logger` to be console, is that desirable? Reason I wanted to keep default null was because this setting will be used in production and I think not logging by default is better than using `console` as default

Reply:
> I think in genera not logging by default is better than using console as default logger

Really? If I was to run something and it not produce logs by default I'd be pretty surprised :open_mouth: !

Reply:
🤔 Hmm, so I was coming from perspective where its possible developer won't know if logs are getting accumulated somewhere as default, and they can add a logger if they want to. But I see your point, think then lets leave it as is with `console` logger as default, anycase for the core usecase this works fine 😄

Reply:
Hmm yeah I see your point! Maybe we could add a `logger.warn` if they use the "default" logger?

Reply:
Sure 👍 This is not a primary usecase(we always inject a logger for core usecase) so think we shouldn't spend too much time on it, and `logger.warn` is good as default, can always update later if there are issues. 😄